diskmaker/controllers/lv/reconcile.go

```diff
@@ -87,11 +88,11 @@ func (r *LocalVolumeReconciler) createSymlink(
 	symLinkSource string,
 	symLinkTarget string,
 	idExists bool,
-) bool {
+) (bool, bool) {
```

Comment:
Can you change these bool return values to a single return value using an enum? I wasn't sure what each one meant until I looked where this function was called: `shouldCreatePV, symlinkCreated`.

There are only 3 combinations that this function returns, so something like this could work:
```
type SymlinkStatus int

const (
    NotCreated SymlinkStatus = 0 // false, false
    SymlinkExists SymlinkStatus = 1 // true, false
    SymlinkCreated SymlinkStatus = 2 // true, true
)
```